pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -62,7 +73,7 @@ func MakeRestorePointsQueryReconciler(r *VerticaRestorePointsQueryReconciler, vr
 
 func (q *QueryReconciler) Reconcile(ctx context.Context, req *ctrl.Request) (ctrl.Result, error) {
 	// no-op if QueryComplete is true
-	isSet := q.Vrpq.IsStatusConditionTrue(vapi.Querying)
+	isSet := q.Vrpq.IsStatusConditionTrue(vapi.QueryComplete)
```

Comment:
@spilchen This is the culprit that causes us to endlessly loop into verticadb.log. `Vapi.Querying` is updated to false to clear the querying status condition. It will no longer be true. I just updated to `vapi.QueryComplete`, and I don't see the endlessly looped behavior anymore